src/core/src/main/java/org/apache/jmeter/threads/AbstractThreadGroup.java

```diff
@@ -86,6 +87,8 @@ public abstract class AbstractThreadGroup extends AbstractTestElement
 
     private final AtomicInteger numberOfThreads = new AtomicInteger(0); // Number of active threads in this group
 
+    private final AtomicLong startTime = new AtomicLong(0);
```

Comment:
Can we use a regular `long` here? If we need to synchronize get/set calls, then we should probably document it

Reply:
I thought more about read semantics in multithread environment. I thought of using a volatile long instead. Would you like that more?

Reply:
That may be true for the thread group, but is true for the timers?

Reply:
I guess the sequence is:
1) Thread group initializes
2) Start time is set
3) Thread group starts
4) Timers operate

The timers should not modify "start time", so there's no need to synchronize the access.